core/src/main/java/com/google/zxing/datamatrix/encoder/HighLevelEncoder.java

```diff
@@ -295,32 +295,34 @@ static int lookAheadTest(CharSequence msg, int startpos, int currentMode) {
         int[] intCharCounts = new int[6];
         byte[] mins = new byte[6];
         findMinimums(charCounts, intCharCounts, Integer.MAX_VALUE, mins);
-        int minCount = getMinimumCount(mins);
 
-        if (intCharCounts[ASCII_ENCODATION] < intCharCounts[BASE256_ENCODATION]
-            && intCharCounts[ASCII_ENCODATION] < intCharCounts[C40_ENCODATION]
-            && intCharCounts[ASCII_ENCODATION] < intCharCounts[TEXT_ENCODATION]
-            && intCharCounts[ASCII_ENCODATION] < intCharCounts[X12_ENCODATION]
-            && intCharCounts[ASCII_ENCODATION] < intCharCounts[EDIFACT_ENCODATION]) {
+        if (intCharCounts[ASCII_ENCODATION] < min(intCharCounts[BASE256_ENCODATION]
+            , intCharCounts[C40_ENCODATION], intCharCounts[TEXT_ENCODATION], intCharCounts[X12_ENCODATION]
```

Comment:
commas on previous line

I tend to write

```
if (foo ||
    (bar &&
     baz)) {
```

Reply:
So do I, but the original code was like that. I will change it.

Reply:
You can leave or change whether the operator is on a new line - whatever looks better to you and is consistent